spec/components/middleware/request_tracker_spec.rb

```diff
@@ -273,6 +273,45 @@ def warn(*args)
       end
     end
 
+    describe "register_static_ip_skipper" do
```

Comment:
The description is slightly confusing here because we're not describing the context of the tests here.